lib/telemetry/instrumenter.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Uniris.Telemetry.Instrumenter do
```

Comment:
I think we can leverage TelemetryMetrics here, with a given supervisor to support the right reporter and the list of metrics to support.

Reply:
Not sure exactly what you mean, but I think I did it in the correct fashion. I added a polling telemetry events example into the supervision tree. Take a look.

Reply:
Ok good now